<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>musicwindow</class>
 <widget class="QMainWindow" name="musicwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>920</width>
      <height>520</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame#frame
{
	image: url(:/new/prefix1/images/b3.jpg);
background-color: transparent;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="back_bt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#back_bt
{
background-color: transparent;
	border-image: url(:/new/prefix1/images/quit.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="title_lab_1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 15pt &quot;微软雅黑 Light&quot;;</string>
     </property>
     <property name="text">
      <string>咪咕音乐</string>
     </property>
    </widget>
    <widget class="QLabel" name="title1_lab_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>421</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>音乐列表</string>
     </property>
    </widget>
    <widget class="QListWidget" name="list_music_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#list_music_2
{
	background-color: transparent;

border: 0px;
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upbtn_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>420</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#upbtn_2
{
	background-color: transparent;
	border-image: url(:/new/prefix1/images/pre2.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="playbtn_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#playbtn_2
{
	border-image: url(:/new/prefix1/images/play2.png);
    background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="down_btn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>420</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#down_btn
{
	border-image: url(:/new/prefix1/images/next2.png);
    background-color: transparent;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="volume">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>430</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/images/volume.png);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="vol_change">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>430</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider#vol_change::groove:horizontal {  
border: 1px solid #4A708B;  
background: #575757;
height: 5px;  
border-radius: 1px;  
padding-left:-1px;  
padding-right:-1px;  
}  
  
QSlider#vol_change::sub-page:horizontal {  
background: qlineargradient(x1:0, y1:0, x2:0, y2:1,   
    stop:0 #B1B1B1, stop:1 #c4c4c4);  
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,  
    stop: 0 #5DCCFF, stop: 1 #1874CD);  
border: 1px solid #4A708B;  
height: 10px;  
border-radius: 2px;  
}  
  
QSlider#vol_change::add-page:horizontal {  
background: #575757;  
border: 0px solid #777;  
height: 10px;  
border-radius: 2px;  
}  
  
QSlider#vol_change::handle:horizontal   
{  
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5,   
    stop:0.6 #45ADED, stop:0.778409 rgba(255, 255, 255, 255));  
  
    width: 11px;  
    margin-top: -3px;  
    margin-bottom: -3px;  
    border-radius: 5px;  
}  
  
QSlider#vol_change::handle:horizontal:hover {  
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0.6 #2A8BDA,   
    stop:0.778409 rgba(255, 255, 255, 255));  
  
    width: 11px;  
    margin-top: -3px;  
    margin-bottom: -3px;  
    border-radius: 5px;  
}  
  
QSlider#vol_change::sub-page:horizontal:disabled {  
background: #00009C;  
border-color: #999;  
}  
  
QSlider#vol_change::add-page:horizontal:disabled {  
background: #eee;  
border-color: #999;  
}  
  
QSlider#vol_change::handle:horizontal:disabled {  
background: #eee;  
border: 1px solid #aaa;  
border-radius: 4px;  
}  </string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
